force-app/main/default/classes/StaleJobApplicationProcess.cls

```diff
@@ -0,0 +1,44 @@
+/**
+ * @description       : 
+ * @author            : Riley Meyer
+ * @group             : 
+ * @last modified on  : 07-27-2024
+ * @last modified by  : Riley Meyer
+**/
+public with sharing class StaleJobApplicationProcess implements Schedulable {
+
+    /**
+    * You can run these lines of code in Developer Console > Apex Anonymous to schedul this Class to run nightly at 1 AM
```

Comment:
Spelling mistake on schedul. Should be schedule.

Reply:
@triden92  should be fixed.